spaceship.zsh

```diff
@@ -16,21 +16,21 @@ NEWLINE='
 # Determination of Spaceship working directory
 # https://git.io/vdBH7
 if [[ -z "$SPACESHIP_ROOT" ]]; then
-  if [[ "${(%):-%N}" == '(eval)' ]]; then
-    if [[ "$0" == '-antigen-load' ]] && [[ -r "${PWD}/spaceship.zsh" ]]; then
-      # Antigen uses eval to load things so it can change the plugin (!!)
-      # https://github.com/zsh-users/antigen/issues/581
-      export SPACESHIP_ROOT=$PWD
-    else
-      print -P "%F{red}You must set SPACESHIP_ROOT to work from within an (eval).%f"
-      return 1
-    fi
-  else
-    # Get the path to file this code is executing in; then
-    # get the absolute path and strip the filename.
-    # See https://stackoverflow.com/a/28336473/108857
-    export SPACESHIP_ROOT=${${(%):-%x}:A:h}
-  fi
```

Comment:
you've got some spacing conflicts too, could you please make sure this whole diff in the PR is as small as possible? 🙂